docs/glossary.md

```diff
@@ -30,4 +30,4 @@
 + **Productions server**: A server on AppSpot. Can be the live server or a staging server.
 + **Spec**: System Specification.
 + **Staging green**: All test are passing against you own deployment on the GAE.
-+ **Staging server**: The server instance you created on AppSpot.
\ No newline at end of file
++ **Staging server**: The server instance you created on AppSpot.
```

Comment:
The problems seems to be just an addition of EOL. Then I'll close one eye for now since it's minor.
That aside, chances are all your text editors have the automatic EOL insertion configured. GitHub editor is one such (and sadly cannot be disabled).